tests/modules/asset_groups/test_models.py

```diff
@@ -79,19 +79,38 @@ def test_asset_group_sightings_jobs(flask_app, db, admin_user, test_root, reques
         str(job_id1): {
             'model': 'african_terrestrial',
             'active': True,
-            'start': now,
+            'start': now.isoformat(),
             'asset_ids': [str(asset_group.assets[0].guid)],
         },
     }
     assert AssetGroupSighting.query.get(ags2.guid).jobs == {
         str(job_id2): {
             'model': 'african_terrestrial',
             'active': True,
-            'start': now,
+            'start': now.isoformat(),
             'asset_ids': [],
         },
     }
 
+    # not exactly sure why this is None, but we need it not-None
```

Comment:
https://docs.google.com/document/d/11TMq1qzaQxva97M3XYwaEYYUawJ5VsrRXnJvTQR_nG0/edit?pli=1#heading=h.iwuh48f58bzt
If the 'speciesDetectionModel' is ‘None’, all AssetGroupSightings will move to the curation stage  for the user to manually create the Annotations.

Reply:
ah ok.  good to know.   me setting _some_ value here is just to get my `get_pipeline_status()` to test what i want... so i dont think this should mess anything up (with respect to changing `speciesDetectionModel`) as the test just ends after this and this is never used (otherwise).